templates/src/csharp/Grpc.Core.Tests/project.json.template

```diff
@@ -12,12 +12,13 @@
     },
     "frameworks": {
       "net45": { },
-      "netstandard1.5": {
+      "netcoreapp1.0": {
         "imports": [
           "portable-net45"
```

Comment:
not super clear on how the framework importing is working, I take it this import still works since netcoreapp1.0 implements the netstandard1.5 anyways?